composer.json

```diff
@@ -6,16 +6,24 @@
         {
             "name": "Caleb Porzio",
             "email": "calebporzio@gmail.com"
+        },
+        {
```

Comment:
Who will be responsible for this package, you or @calebporzio ?

Reply:
I run a customized `composer normalize` command - for sure this isn't required in any way.

Reply:
https://github.com/calebporzio/sushi/pull/7/commits/a515972b622342265eab7b08b845ea1047890ba5